xapian-applications/omega/templates/query

```diff
@@ -153,3 +153,4 @@ $map{$cgilist{B},<input type=hidden name=B value="$html{$_}">}
 </body>
 </html>
 $log{query.log}
+$log{search.log,$hash{md5,$transform{s/^\s+|\s+$$//g,\1,$query}}	$csv{$query,1}	$hitlist{$id$.}	$topdoc}
```

Comment:
FWIW, if literal tabs are felt to be problematic, then `$chr{9}` can be used.

Why is CSV-style quoting useful when we're using a tab separator?  Is it essentially that we need to quote the string if it contains literal tabs, and `$csv` in "always-quote" mode achieves that?